src/utils/fetchUtils.ts

```diff
@@ -1,27 +1,34 @@
+import { endOfDay, startOfDay } from "date-fns";
+import { zonedTimeToUtc } from "date-fns-tz";
 import debug from "debug";
 import fetch from "node-fetch";
 import { URL } from "url";
 import {
+  IEvent,
   IGuide,
   ILanguage,
   INavigationCategory,
   IPointProperty,
 } from "../types/typings";
 import { validate } from "./jsonValidator";
 import {
+  parseEvent,
   parseGuide,
   parseGuideGroup,
   parseLanguage,
   parseNavigationCategory,
 } from "./parsingUtils";
 import {
+  buildEventsUrl,
   buildGuideGroupUrl,
   buildGuideUrl,
   buildLanguagesUrl,
   buildNavigationUrl,
   buildPropertyUrl,
 } from "./urlUtils";
 
+const TZ = process.env.TZ || "Europe/Stockholm";
```

Comment:
Should we use envUtils constant instead of accessing `process.env.TZ` directly here?

Reply:
Yeah probably better.
```
import { TZ } from ...envutils
const TZ = process.env[TZ].. 
```
Or did you have something else in mind?

Reply:
That's probably fine for now. It's bit of an overkill to add a service specifically for env in the stage we're at now.